psalm.xml

```diff
@@ -148,6 +148,12 @@
             </errorLevel>
         </UndefinedDocblockClass>
 
+        <UndefinedMethod>
+            <errorLevel type="info">
+                <file name="src/Sylius/Bundle/CoreBundle/DataFixtures/Factory/ShopUserFactory.php" />
```

Comment:
What is the method? I would rather add the annotations inline.

Reply:
`findOrCreate` method
https://github.com/Sylius/Sylius/pull/13543/files#diff-db5036129c448c53aa427624ed5e44a5176ac04fa0d4849a1bf183629a618534R30

Reply:
I don't know why Psalm does not look at the docblock

Reply:
Looks like it only works if the `__call` method is implemented: https://psalm.dev/docs/annotating_code/supported_annotations/#supported-docblock-annotations